Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q -f \
+		./*.md doc src lib benchmark tools/doc/ tools/icu/
```

Comment:
`lib` contains `lib/internal/readme.md` which should be linted.

`src` atm doesn't include markdown files, but it makes sense to keep it in case if those files would be added in the future. There should be no measurable performance impact from including that directory now, as only markdown files are checked and there are none there.